rmf_visualization_schedule/config/rmf.rviz

```diff
@@ -24,7 +24,7 @@ Panels:
     Splitter Ratio: 0.5
   - Class: rviz_common/Selection
     Name: Selection
-  - Class: rviz2_plugin/SchedulePanel
```

Comment:
I'm surprised this works. Shouldn't it be renamed to `rmf_visualization_rviz2_plugins/SchedulePanel`?

Reply:
My bad, missing commit for this. You are right.